elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/SecureDeploymentDefinition.java

```diff
@@ -96,6 +96,12 @@ class SecureDeploymentDefinition extends SimpleResourceDefinition {
                     .setAlternatives(ElytronOidcDescriptionConstants.RESOURCE)
                     .build();
 
+    protected static final SimpleAttributeDefinition SCOPE =
+            new SimpleAttributeDefinitionBuilder(ElytronOidcDescriptionConstants.SCOPE, ModelType.STRING, true)
+                    .setAllowExpression(true)
```

Comment:
Please also update the oidc-expressions.xml test file to configure scopes for a secure-server and a secure-deployment using an expression.

Reply:
Thanks. I noticed the values for the attributes are different inside `oidc-expressions.xml`. Would I replace the values for the new secure server and secure deployment to be consistent with the existing values inside `oidc-expressions.xml`? (i. e. `{exp.provider:redhat-sso-expressions}` for provider instead of `keycloak`?)
Additionally, there are more attributes defined for each deployment in oidc-expressions compared to oidc.xml. Should I specify those additional attribute values from the new deployments as well?

Reply:
The `oidc-expressions.xml` test file ensures that every attribute that can be specified using an expression can be parsed successfully.

We can just add the scope attribute to one of the existing `secure-deployments` in that file and to one of the existing `secure-servers` in that file and set its value to an expression.

When taking a look at the expressions in that file, we can see they are formatted as follows:

`${exp.name:some_default_value}`

Reply:
Alright, I will add scope to an existing one. 
Thanks

Reply:
fixed! Added new scopes to `wildfly-console` and `wildfly-management-expressions.war`